ogr/services/pagure/project.py

```diff
@@ -291,6 +292,20 @@ def exists(self):
         response = self._call_project_api_raw()
         return response.ok
 
+    def is_private(self) -> bool:
+        """
+        Is this repo private? (accessible only by users with granted access)
+
+        :return: if yes, return True
+        """
+        host = urlparse(self.service.instance_url).hostname
```

Comment:
We already have the parsing ability in `./parsing.py` that can be used instead, but  it's not so important.

```python
In [1]: from ogr.parsing import parse_git_repo                                        

In [2]: str(parse_git_repo("https://pagure.io"))                                      
Out[2]: "RepoUrl(repo='', namespace='None', is_fork=False, hostname='pagure.io', scheme='https')"
```